app/assets/stylesheets/components/_button.sass

```diff
@@ -0,0 +1,19 @@
+$Button-default-backgroundColor: Theme-color(button, default)
+$Button-text-color: Theme-color(text, lighter)
+$Button-height: 50px
+$Button-spacing: $Theme-spacing-base
+
+.Button
+  text-align: center
+
+.Button-default
+  background-color: $Button-default-backgroundColor
+  border-radius: 3px
+  border: 1px solid darken($Button-default-backgroundColor, 10%)
+  color: $Button-text-color
+  line-height: $Button-height
+  margin-top: $Button-spacing
+  padding: 10px 20px 10px 20px
```

Comment:
I was making the button, and this values seems nice to me. But yeah, it does not follow any method.
I changed to use the `padding: $Theme-spacing-small $Theme-spacing-small`, sounds good?